plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/utils/ConstructorUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.hotswap.agent.plugin.spring.utils;
+
+import org.springframework.beans.factory.support.RootBeanDefinition;
+import org.springframework.util.ReflectionUtils;
+
+import java.lang.reflect.Method;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+
+/**
+ * copy from org.springframework.beans.factory.support.ConstructorResolver#getCandidateMethods(java.lang.Class, org.springframework.beans.factory.support.RootBeanDefinition)
+ * The method is not public, so we have to copy it. It avoids using the reflection invocation.
+ */
+public class ConstructorUtils {
```

Comment:
ConstructorUtils is not a good name, or should we simply move 'getCandidateMethods' (the only method in this class) into org.hotswap.agent.util.spring.util.ReflectionUtils?